syntax = "proto3";
import "google/protobuf/empty.proto";
import "common.proto";
package gs2gw;
option cc_generic_services = true;

message MessageExtern
{
  uint64 conn_id = 1;
}

message PlayerMessageRequest
{
  MessageBody msg = 1;
  MessageExtern ex = 2;
}

service Gs2GwService 
{
  rpc PlayerService (PlayerMessageRequest) returns (google.protobuf.Empty);
}
